packages/fxa-content-server/package.json

```diff
@@ -155,7 +155,7 @@
         "grunt-todo": "0.5.0",
         "htmlparser2": "3.9.0",
         "install": "0.12.1",
-        "intern": "4.3.1",
+        "intern": "^4.4.3",
         "jsqr": "1.2.0",
         "leadfoot": "1.7.4",
```

Comment:
might need to bump this too here

Reply:
That is the latest leadfoot on npm.  `intern` itself uses `@theintern/leadfoot`, which is much newer.

Reply:
should switch to `@theintern/leadfoot` here then